packages/ckeditor5-list/src/listproperties/ui/listpropertiesview.ts

```diff
@@ -366,6 +369,7 @@ export default class ListPropertiesView extends View {
 			const startIndex = inputElement.valueAsNumber;
 
 			if ( Number.isNaN( startIndex ) ) {
+				startIndexFieldView.errorText = t( 'Invalid start index value.' );
```

Comment:
second-thought question: do we have any core translation for just 'invalid format' or something like this? we could avoid the new key.